src/main/java/by/itechart/api/dto/UserDTO.java

```diff
@@ -5,10 +5,9 @@
 @NoArgsConstructor
 @Data
 public class UserDTO {
-    Long id;
```

Comment:
As I remember we agreed to create a new model class for user update data, and leave this one to display data in the list. In that case, we need to have ID here too, so I would recommend you to not remove it.

Reply:
Yeah, you are right :3 I will create a new DTO for that.